package.json

```diff
@@ -166,7 +166,7 @@
     "tinyliquid": "^0.2.33",
     "url-loader": "^0.5.7",
     "uswds": "^1.0.0",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema#7ef76dfb7e334a23db5b663a8c0dff15747748f6",
+    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema#32580687bb7308e638a1d6ef675ca7a92cc4cd35",
```

Comment:
I have it figured out, but I have to make another schema change, so this won't pass quite yet.